packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/actionButtons.test.js

```diff
@@ -1,1065 +0,0 @@
-import $ from 'jquery';
```

Comment:
Why did you delete these tests?

Reply:
I was going to rename it to 'tests.js' to match the naming of the other files, but something went wrong. Well, it doesn’t matter here, so I reverted the change.